### PR TITLE
[MRG] Added pygbm to lightgbm model conversion

### PR DESCRIPTION
**Pull request by NicolasHug**
Created `utils.py` with a utility that builds a lightgbm estimator with the same hyperparameters of a pygbm estimator.

### CODE REVIEW

File: pygbm/utils.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
+from lightgbm import LGBMRegressor
+from lightgbm import LGBMClassifier
+
+from .gradient_boosting import GradientBoostingClassifier
+
+
+def get_lightgbm_estimator(pygbm_est):
+    """Return an estimator from LightGBM that has the same hyperparameters
+    as pygbm_est"""
```
Comment (ogrisel) [inline pygbm/utils.py]:
```python
"""Return an unfitted LightGBM estimator with matching hyperparams

This utility function takes care of renaming the PyGBM parameters into
their LightGBM equivalent parameters.

The PyGBM estimator is assumed to be unfitted (no trained trees).
"""
```

File: pygbm/utils.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
+from lightgbm import LGBMRegressor
+from lightgbm import LGBMClassifier
+
+from .gradient_boosting import GradientBoostingClassifier
+
+
+def get_lightgbm_estimator(pygbm_est):
```
Comment (ogrisel) [inline pygbm/utils.py]: `pygbm_est` => `pygbm_estimator`

File: pygbm/utils.py [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    """Return an estimator from LightGBM that has the same hyperparameters
+    as pygbm_est"""
+
+    pygbm_params = pygbm_est.get_params()
+
+    if pygbm_params['loss'] == 'auto':
+        raise ValueError('auto loss is not accepted. We need to know if '
+                         'the problem is binary or multiclass classification.')
+    if pygbm_params['scoring'] is not None:
+        raise ValueError('Early stopping should be deactivated.')
```
Comment (ogrisel) [inline pygbm/utils.py]: Maybe raise `NotImplementedError` instead?

**Comment by NicolasHug** [comment]
Thanks for the review @ogrisel

**Pull request merged**